articles/active-directory/develop/quickstart-v2-javascript.md

```diff
@@ -117,6 +111,29 @@ Select the option that's suitable to your development environment:
 > > To find the values of **Application (client) ID**, **Directory (tenant) ID**, and **Supported account types**, go to the app's **Overview** page in the Azure portal.
 >
 
+> [!div renderon="docs"]
+> Then, still in the same folder, edit *graphConfig.js* file to set the `graphMeEndpoint` and `graphMeEndpoint` for the `apiConfig` object.
+
+```javascript
+
+  // Add here the endpoints for MS Graph API services you would like to use.
+  const graphConfig = {
+    graphMeEndpoint: "Enter_the_Graph_Endpoint_Herev1.0/me",
+    graphMailEndpoint: "Enter_the_Graph_Endpoint_Herev1.0/me/messages"
+  };
+
+  // Add here scopes for access token to be used at MS Graph API endpoints.
+  const tokenRequest = {
+      scopes: ["Mail.Read"]
+  };
+
```

Comment:
Ln 117-128 should be docs only. Code blobs for copy-pasting are no longer needed in the portal.